### PR TITLE
Triangles per vertex

### PR DESCRIPTION
**Pull request by ep09772**
stats and tests for triangles per vertex

### CODE REVIEW

File: dagmc_stats.py [hunk 1]
```diff
@@ -84,10 +84,32 @@ def get_entityset_ranges(my_core, meshset, geom_dim):
...
+    for vertex in native_ranges[0]:
+        t_p_v_data.append(my_core.get_adjacencies(vertex, tri_dimension).size())
+    return t_p_v_data
+  
+  
 def get_triangles_per_surface(my_core, entity_ranges):
     """
     This function will return data about the number of triangles on each
-    surface in a file
+    vertex in a file
```
Comment (gonuke) [inline dagmc_stats.py]: I think this changes it the wrong way.

Comment (ep09772) [reply dagmc_stats.py]: Oops

File: dagmc_stats.py [hunk 2]
```diff
@@ -84,6 +84,28 @@ def get_entityset_ranges(my_core, meshset, geom_dim):
...
+    native_ranges : a dictionary containing ranges for each native type in the file (VERTEX, TRIANGLE, ENTITYSET)
+    
+    outputs
+    -------
+    t_p_v_data : a list of the number of triangles each vertex touches
+    """
+    
+    t_p_v_data = []
+    tri_dimension = 2
+    for vertex in native_ranges[0]:
```
Comment (gonuke) [inline dagmc_stats.py]:
```suggestion
    for vertex in native_ranges[types.MBVERTEX]:
```

Comment (ep09772) [reply dagmc_stats.py]: fixed

**Comment by gonuke** [comment]
Thanks @ep09772

**Pull request merged**